static/scss/search-page.scss

```diff
@@ -365,3 +365,20 @@
     color: rgba(0,0,0,0.8);
   }
 }
+
+.sk-hierarchical-menu-list {
+  width: 100%;
+
+  .sk-hierarchical-menu-list__root {
+    margin-left: 10px;
+    margin-right: 10px !important;
```

Comment:
Agreed. Using `!important` is a "code smell": it indicates that something is probably incorrect. A comment helps to justify why `!important` is actually necessary. If you can't find a good reason for using it, don't use it!